packages/playwright-core/src/cli/cli.ts

```diff
@@ -53,13 +53,18 @@ const kExternalPlaywrightTestCommands = [
   ['show-report', 'Show Playwright Test HTML report.'],
   ['merge-reports', 'Merge Playwright Test Blob reports'],
 ];
-for (const [command, description] of kExternalPlaywrightTestCommands) {
-  const playwrightTest = program.command(command).allowUnknownOption(true);
-  playwrightTest.description(`${description} Available in @playwright/test package.`);
-  playwrightTest.action(async () => {
-    printPlaywrightTestError(command);
-    gracefullyProcessExitDoNotHang(1);
-  });
+function addExternalPlaywrightTestCommands() {
+  for (const [command, description] of kExternalPlaywrightTestCommands) {
+    const playwrightTest = program.command(command).allowUnknownOption(true);
+    playwrightTest.description(`${description} Available in @playwright/test package.`);
+    playwrightTest.action(async () => {
+      printPlaywrightTestError(command);
+      gracefullyProcessExitDoNotHang(1);
+    });
+  }
 }
 
+if (!process.env.PW_LANG_NAME)
```

Comment:
`|| === 'javascript'` - just in case we start running the driver out of process?

Reply:
So far we use 'not set' as a good default.

Reply:
Yeah, whoever touches it will never know that this place need to be updated too.